com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/AtlasUtil.java

```diff
@@ -86,10 +103,10 @@ public AnimDesc nextAnim() {
         }
 
         @Override
-        public Integer nextFrameIndex() {
+        public Integer nextFrameIndex() { // Return the global index of the image that the frame is using
             MappedAnimDesc anim = anims.get(nextAnimIndex - 1);
-            if (nextFrameIndex < anim.getIds().size()) {
-                return imageIds.indexOf(anim.getIds().get(nextFrameIndex++));
+            if (nextFrameIndex < anim.getPaths().size()) {
+                return imagePaths.indexOf(anim.getPaths().get(nextFrameIndex++));
```

Comment:
Lookup using the unique project path for the image resource (.png), giving us the index into the image data in the atlas.